app/javascript/packages/document-capture/components/in-person-prepare-step.tsx

```diff
@@ -92,6 +95,17 @@ function InPersonPrepareStep({ toPreviousStep, value }) {
           </Button>
         </div>
       )}
+      <p>
+        {t('in_person_proofing.body.prepare.privacy_disclaimer')}{' '}
+        {securityAndPrivacyHowItWorksURL && (
```

Comment:
> I'm curious about why this is a conditional. Would it ever be falsy?

In this case, no. I was on the fence about it. The main thing for me was envisioning that `MarketingSiteContext` could be something we'd want to extract from `document-capture` and use elsewhere in the application, and imposing that the `securityAndPrivacyHowItWorksURL` would be a required prop for it might be overly restrictive, so I decided to make it optional. Since the `<Link>` component requires a string for `href`, we need to handle the optional-ness. We could use [the non-null assertion operator](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-) to indicate that we are certain it won't be `undefined`, but I've been trying to avoid using it too much.

On the other hand, it did make things a bit more complicated to manage here 😄

Reply:
To answer your question more directly, yes it can technically be falsey due to the context type, even though we don't expect it to be in how we're setting up the page.

https://github.com/18F/identity-idp/blob/7d07e14c80901600dccc2540616d3bed75d1a3da/app/javascript/packages/document-capture/context/marketing-site.tsx#L19